### ISSUE TITLE
Archived closed issues/PRs

### ISSUE THREAD

**Issue by mirkopoloni** [volta-net/feedback#12]
When a PR has been closed, there is the possibility to archive it (see attached screenshot).

Where can I find the archived element again? I wasn't able to find it.
The problem is that if the PR will be reopened again on the GitHub website, it still won't appear back in the Volta app.
<img width="1077" alt="Screen Shot 2022-10-21 at 12 45 20" src="https://user-images.githubusercontent.com/32099118/197178920-cb568413-2ead-482d-a7a8-639d91c58b58.png">

**Comment by benjamincanac** [comment]
Hello @mirkopoloni,

At this moment the only way to display archived issues is to go to the specific url `/:owner/:name/issues/:number`. Then, from the issue page you can archive / unarchive:

![CleanShot 2022-10-21 at 15.17.24@2x.png](https://volta.s3.fr-par.scw.cloud/Clean_Shot_2022_10_21_at_15_17_24_2x_57eedac60c.png)

We're working on a way to display those directly from the board, but it might take some time as we have lots of priorities.

However, when an issue / pr is reopened it should be automatically unarchived. If this is not the case, could you provide more informations if the concerned issue is public?

**Comment by mirkopoloni** [comment]
Hello @benjamincanac and thanks for the quick answer.

Navigating to the url worked perfectly 👍 
This way I am also able to navigate to previously merged/closed PRs which are not visible in the board.
Is it normal that all closed PRs before the registration to Volta are not visible?

I tried to reopen the PR but it doesn't get unarchived. I tested it in a private repository and I can't share it.
Is there another way how I could help you? Maybe sending a video with the problem?

**Comment by benjamincanac** [comment]
It is normal that closed issues and pull requests that were not affected to a status (closed prior import on Volta) don't show up in your boards. It would load an enormous amount of data otherwise and wouldn't benefit from the automation as their flow is done.

Are you sure the PR you're talking about is really archived? It might be just not returned by our API. You can try to put it in `Done` status for example to have it on your board.

**Comment by mirkopoloni** [comment]
Hey @benjamincanac, sorry for the late answer.
I made a video to reproduce the problem I am facing:
1 - Created a new PR
2 - Moved to cancelled in Volta
3 - PR was closed on GitHub as expected
4 - Archived PR in Volta
5 - Reopened PR on GitHub
6 - PR is not visible in Volta, maybe it doesn't get unarchived somehow?

Am I doing something wrong? Is it the expected behaviour?

https://user-images.githubusercontent.com/32099118/212542670-9b4617bf-9781-41b5-b793-7263fe035c98.mov


https://user-images.githubusercontent.com/32099118/212542651-ec578595-805c-4756-9afe-3945f2b937ad.mov


https://user-images.githubusercontent.com/32099118/212542661-667bb835-f1b6-43bd-8e11-40ce559b91c7.mov
